types/http-errors/http-errors-tests.ts

```diff
@@ -91,9 +87,16 @@ err = new create[404]('This might be a problem');
 err = new create.MisdirectedRequest();
 err = new create.MisdirectedRequest('Where should this go?');
 
+// $ExpectType HttpError
+new create.HttpError();
+
 // $ExpectType boolean
 new Error() instanceof create.HttpError;
 
```

Comment:
`err` is already a `HttpError` here, I'm not sure if it matters but we could potentially do something like:

```typescript
const genericError = new Error('test')

if (genericError instanceof create.HttpError) {
  genericError.statusCode
}
```